deps/rabbitmq_management/priv/www/js/main.js

```diff
@@ -22,27 +22,63 @@ function startWithOAuthLogin () {
   store_pref("oauth-return-to", window.location.hash);
 
   if (!oauth.logged_in) {
-    if (oauth.sp_initiated) {
-      get(oauth.readiness_url, 'application/json', function (req) {
+
+    if (oauth.resource_servers.length == 1 && oauth.resource_servers[0].sp_initiated) {
+      get(readiness_url(oauth.resource_servers[0]), 'application/json', function (req) {
         if (req.status !== 200) {
-          renderWarningMessageInLoginStatus(oauth.authority + ' does not appear to be a running OAuth2.0 instance or may not have a trusted SSL certificate')
+          let message = 'Unable to retrieve OpenID configuration from ' + oauth.resource_servers[0].oauth_provider_url + '. '
+          switch(req.status) {
+            case 0:
+              message += 'Reason: ' + oauth.resource_servers[0].oauth_provider_url + " not reachable"
+              break
+            case 404:
+              message += 'Reason: ' + readiness_url(oauth.resource_servers[0]) + " return 404"
+              break
+            default:
+              message += 'Reason: ' + req.statusText
+          }
+          console.log(message)
```

Comment:
Do we want to keep the `console.log` statements or are they leftovers from development?

Reply:
yes, i want to keep this one and others to help troubleshoot the reason why it failed. To be honest, it is a bit redundant because this same message is rendered in the ui . but just in case ..